mbed_lstools/lstools_base.py

```diff
@@ -48,7 +51,7 @@ class MbedLsToolsBase:
     MBED_HTM_NAME = 'mbed.htm'
     ERRORLEVEL_FLAG = 0
 
-    def __init__(self, **kwargs):
+    def __init__(self, list_unmounted=False, **kwargs):
```

Comment:
@theotherjimmy this should probably be in `list_mbeds`, not in the constructor, wouldn't you say? Seems a bit strange to have to reconstruct the object to change this option, especially since it's only used by the `list_mbeds` function.

Reply:
I'm only changing internal APIs in this PR. This is an external API.

Reply:
According to the diff you've added an optional parameter here. So adding it to `list_mbeds` wouldn't be a disruptive change. It'd be a "minor" bump vs a "major"

Reply:
Actually, this parameter is already accepted. I just made it explicit. It was parsed from `kwargs` before.

Reply:
Ah I see. Ok, it makes more sense to me for it being in `list_mbeds`, but we could always revisit that. Don't need to hold up this PR and all of its other goodness.